optaplanner-core/src/test/java/org/optaplanner/core/impl/domain/solution/descriptor/SolutionDescriptorTest.java

```diff
@@ -195,6 +203,20 @@ public void extendedAbstractSolutionOverridesGetScore() {
         assertAllCodesOfCollection(solutionDescriptor.getAllFacts(solution), "e1", "e2", "v1", "v2", "extra");
     }
 
+    // ************************************************************************
+    // Autodiscovery
```

Comment:
Just a comment (not needed to do this): It implies to me that if some tests are marked with such header, they could be part of their own test class. E.g. in this case SolutionDescriptorAutodiscoveryTest.